src/default_env_vars.yml

```diff
@@ -6,6 +6,7 @@ ALLIANCE_RELEASE: "0.0.0"
 NEO4J_HOST: "localhost"
 NEO4J_PORT: 7687
 FMS_API_URL: "https://fms.alliancegenome.org"
+HEADER_TEMPLATE_URL: "https://raw.githubusercontent.com/alliance-genome/agr_file_generator/6fea7c7a12ef8f53de17f31f8a8bd3b468cb41e7/src/generators/header_template.txt"
```

Comment:
Will this go somewhere more permanent at some point? Just curious?

Reply:
Valerio's just made these changes https://github.com/alliance-genome/agr_file_generator/pull/85 to create a pip module for the agr_file_generator repo, so that we can generate the headers with it.  I may add this to preprocess for 3.1, but the headers should already be in the same format, so I may wait until the next release, since we'll have to talk to people and curators about how the headers should be formed.  (Right now if the headers format changes slightly, we'd just need to change one variable in the .yml file, if we change it to the module, I think we'd have to change the requirements file + the imports in the .py files that use it.  If the headers change a lot the module would be better though)